dm/syncer/syncer.go

```diff
@@ -1875,7 +1875,28 @@ func (s *Syncer) Run(ctx context.Context) (err error) {
 						// revert currentLocation to startLocation
 						currentLocation = startLocation
 					} else if op == pb.ErrorOp_Skip {
+						ec := eventContext{
+							tctx:            tctx,
+							header:          e.Header,
+							startLocation:   &startLocation,
+							currentLocation: &currentLocation,
+							lastLocation:    &lastLocation,
+						}
+						var sourceTbls map[string]map[string]struct{}
+						sourceTbls, err = s.trackOriginDDL(ev, ec)
```

Comment:
if user use `binlog-schema update` to set schema in tracker, and then use `binlog skip`, we maybe should note that original ddls will be still execute. just a slight worry, i don't find error conditions.